.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - php: 8.1
```

Comment:
Can you also add a new option like entry like this one? https://github.com/B-Galati/monolog-sentry-handler/blob/main/.github/workflows/ci.yml#L17

It would ensure our library works well with the lowest version possible.

Reply:
I saw this line reviewing the MR 👍🏻